app/components/provider_interface/organisation_list_component.rb

```diff
@@ -0,0 +1,21 @@
+module ProviderInterface
+  class OrganisationListComponent < ViewComponent::Base
+    attr_accessor :training_permissions, :ratifying_permissions, :current_provider_user
+
+    def initialize(provider:,  current_provider_user:)
+      @current_provider_user = current_provider_user
+      @training_permissions = ProviderRelationshipPermissions.where(training_provider: provider.id)
+        .or(ProviderRelationshipPermissions.where(training_provider: all_manageable_providers, ratifying_provider: provider))
+        .where.not(setup_at: nil)
+        .includes(:training_provider)
+      @ratifying_permissions = ProviderRelationshipPermissions.where(ratifying_provider: provider.id).where.not(training_provider: all_manageable_providers)
+        .includes(:training_provider, :ratifying_provider)
```

Comment:
minor adjustment so it matches the format of `@training_permissions`
```suggestion
      @ratifying_permissions = ProviderRelationshipPermissions.where(ratifying_provider: provider)
        .where.not(training_provider: all_manageable_providers)
        .includes(:training_provider, :ratifying_provider)
```